### DISCUSSION TITLE
Connect to tables in other schemas (not managed by directus)

### DISCUSSION THREAD

**Discussion by geoHeil** [directus/directus#18056] [Archive]
Is it possible to connect/interface a directus collection with other tables managed outside/externally of directus? If yes, how can this be enabled?

**Comment by rijkvanzanten** [comment]
No(t yet)

**Reply by geoHeil** [reply]
Is there an issue/feature request to track for this?

**Reply by rijkvanzanten** [reply]
Not to my knowledge!
